health-sdk/sdk/src/test/java/net/gini/android/health/sdk/paymentComponent/PaymentComponentTest.kt

```diff
@@ -502,4 +518,58 @@ class PaymentComponentTest {
 
         paymentComponentPreferences.deleteSelectedPaymentProviderId()
     }
+
+    @Test
+    fun `rechecks returning user`() = runTest {
+        // Given
+        val paymentComponent = PaymentComponent(context!!, giniHealth!!)
+//        val paymentComponentPreferences = PaymentComponentPreferences(context!!)
```

Comment:
[Request] Please remove commented out lines.

Reply:
Thank you for catching this! Removed commented code.